quantum-viz/examples/Qiskit.ipynb

```diff
@@ -0,0 +1,339 @@
+{
```

Comment:
This notebook seems more like a scratchpad that you were using for testing during development that an actual example on how to use the new qiskit integration. I'll send you a PR with a new version that is better suited as an example.

Reply:
Hi, this is actually not even mine, it is transferred from #36.
I do not mind removing that.

Reply:
Being addressed in jond01#4

Reply:
Done in jond01#4 + jond01#6